mbtiles/scripts/cli.py

```diff
@@ -14,70 +14,162 @@
 from rasterio.rio.options import overwrite_opt, output_opt
 from rasterio.warp import transform
 
-from mbtiles import init_worker, process_tile
 from mbtiles import __version__ as mbtiles_version
 
 
-DEFAULT_NUM_WORKERS = cpu_count() - 1
+DEFAULT_NUM_WORKERS = None
```

Comment:
Using one fewer workers than the number of available CPUs seems like a friendlier default option than taking all available CPUs.

Reply:
I'm inclined to trust the Python defaults here. We do allow user control over the number of workers.

Reply:
Okay, that makes sense. I recall regretting the decision to manually set all four of my CPUs as available to some python processes in the past on my old laptop, and my read of what `concurrent.futures` did was it would grab all available CPUs, but I don't know that it isn't doing under the hood.